src/unread/unreadPmsReducer.js

```diff
@@ -46,7 +47,32 @@ const eventUpdateMessageFlags = (state, action) => {
   if (action.op === 'add') {
     return removeItemsDeeply(state, action.messages);
   } else if (action.op === 'remove') {
-    // we do not support that operation
```

Comment:
nit: the commit message says "unreads: Add support for marking PMs as unread", but it doesn't -- it only adds support for 1:1 PMs, not group PMs.

See e.g. user-facing docs:
https://zulip.com/help/reading-pms
where we say "private messages (PMs)" to mean both kinds of PMs together.

In general we're consistent about that in the UI and user-facing docs, as well as in like 95% of the Zulip codebase. Even in parts of the server where a "huddle" is an important concept, if we say "private message" or "PM" we almost always mean a message that's either to a huddle or an individual user, not only the latter. E.g. search zerver/models.py for /private.message/i.

(I know that this `src/unread/` section of the codebase often says "PM" to mean just 1:1 PMs. That's an unfortunate exception, but we can still use the standard terminology in commit messages.)